CMakeLists.txt

```diff
@@ -9,6 +9,11 @@ project(freeopcua)
 
 option(BUILD_CLIENT "Build Client" ON)
 option(BUILD_SERVER "Build Server" ON)
+
+option(SSL_SUPPORT_MBEDTLS "Support rsa-oaep password encryption using mbedtls library " OFF)
+option(SSL_SUPPORT_MBEDTLS_INCLUDE_DIR "mbedtls include dir " "/usr/local/include")
+option(SSL_SUPPORT_MBEDTLS_LIB_DIR "mbedtls libraries dir " "/usr/local/lib")
```

Comment:
doesn't have cmake a way to find mbedtls? I am quite sure there are ways to avoid hard coding including path